benchmarks/bfv_zkp/src/bfv.rs

```diff
@@ -260,16 +260,9 @@ pub fn compile_proof() -> ZkpApplication {
 fn ark_bigint_to_native_field<B: ZkpBackend, F: MontConfig<N>, const N: usize>(
     x: Fp<MontBackend<F, N>, N>,
 ) -> NativeField<B::Field> {
-    assert!(N <= 8);
```

Comment:
We should keep some assertion here, as converting a bigint is only correct if the Arkworks int <= 512 bit, regardless of the limb size.

Maybe:
```
let max_limbs = if cfg!("32_bit") {
  16
} else if cfg!("64_bit) {
  8
} else {
  unreachable!();
}

assert!(N <= max_limbs);
```

and then somewhere else doing
```
#[cfg(not(any("32_bit", "64_bit"))))]
compile_error!("Unsupported limb width");
```